main.go

```diff
@@ -37,4 +71,41 @@ func main() {
 	if err := vmm.Boot(); err != nil {
 		log.Fatal(err)
 	}
+
+	return nil
+}
+
+type debugCmd struct{}
+
+func (d *debugCmd) Run(_ *context) error {
+	if err := probe.KVMCapabilities(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+type cli struct {
+	Start startCmd `cmd:"" help:"Starts a new VM"`
+	Debug debugCmd `cmd:"" help:"Prints KVM capabilities"`
+}
+
+func main() {
+	c := cli{}
+
+	programName := "gokvm"
+	programDesc := "gokvm is a small Linux KVM Hypervisor which supports kernel boot and XEN PVH boot protocol"
```

Comment:
PVH boot protol is not supported yet in main branch. So could you remove for now?
```suggestion
	programDesc := "gokvm is a small Linux KVM Hypervisor which supports kernel boot"
```

Reply:
done